### ISSUE TITLE
`MA0051` Should not warn with a huge switch expression

### ISSUE THREAD

**Issue by RenderMichael** [meziantou/Meziantou.Analyzer#680]
### Version of the Meziantou.Analyzer NuGet package

2.0.135

### Rule Identifier

MA0051

### Target Framework

net8.0

### C# Language version

C# 12

### Description

I  have some methods that are > max lines but they are large switch expressions. This seems like a false positive to me, as that is my intended design.

### Reproduction Steps

Minimal code:

```c#
 public static string GetExtension(string? mimeType) => mimeType switch
 {
     "application/andrew-inset" => ".ez",
     "application/applixware" => ".aw",
     "application/atom+xml" => ".atom",
     "application/atomcat+xml" => ".atomcat",
     "application/atomsvc+xml" => ".atomsvc",
     "application/ccxml+xml" => ".ccxml",
     ... // on and on for >max lines
}
```

**Comment by meziantou** [comment]
I think you should configure the maximum number of statements per method instead of lines. See https://github.com/meziantou/Meziantou.Analyzer/blob/main/docs/Rules/MA0051.md for more info. In your example, there is only one statement.
